### DISCUSSION TITLE
About these small dividers (screenshot)

### DISCUSSION THREAD

**Discussion by ContinuumOperand** [Devolutions/UniGetUI#2040] [General]
![image](https://github.com/marticliment/WingetUI/assets/145922748/f82b47d1-1b57-43e0-9e9f-25a54f6c1c01)

Are they really necessary? They feel out of place/unnecessary for me, but I'm imagining you wanted a way a separate the different group buttons, right?

Would it be possible to just use additional spacing to accomplish that instead or to have a background highlight/outline spanning across the respective group of buttons?

Just my thoughts, cheers!

*Sorry if it sounds nit-picky, haha.*

**Comment by marticliment** [comment]
IMO, they help separate the buttons into categories. However I will do dome testing on higher highlights as suggested

**Reply by ContinuumOperand** [reply]
Thanks for quick response. I appreciate you looking into it. I think UI zoning/highlights with 'elevation' based on color brightness is a good design, in my eyes at least.

**Reply by ContinuumOperand** [reply]
The way I saw them is that the icons are already self-explanatory enough, to be 'intued' from the active tab context as a user, but I understand there will be different levels of accessibility and 'ease' of use for different people.

**Comment by Spacellary** [comment]
@marticliment – Hey! I'm just curious if you had the chance to prototype anything on this or any other thoughts. Stay awesome.

**Reply by marticliment** [reply]
Not yet, but I have been looking at other WinUI3 apps, and they all seem to use the same separators, so I may as well leave them how they are by default

**Reply by Spacellary** [reply]
It's probably just my weird OCD, somehow they seem out of place with other elements using Color/Brightness Depth but...
